templates/vue/src/components/tyde/activities/CircleOfSupport/CommunityView/onboarding/AddConfirmation.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <cos-modal show @change="$emit('later')">
+    <b-container class="confirm">
+      <b-row align-h="center">
+        <h2 class="ob-title">Great job! Here are all your communities.</h2>
+      </b-row>
+      <b-row align-h="center">
+        <h4 class="ob-secondary">Would you like to add another community?</h4>
+      </b-row>
+      <b-row>
+        <b-container>
+          <b-row class="controls">
+            <b-col>
+              <b-button
+                pill
+                variant="secondary"
+                class="secondary"
+                @click="$emit('later')"
+              >
+                Maybe later &#8594;
+              </b-button>
+            </b-col>
+            <b-col>
+              <b-button
+                pill
+                variant="secondary"
+                class="primary"
+                @click="$emit('another')"
+              >
+                Add another &#8594;
+              </b-button>
+            </b-col>
+          </b-row>
+        </b-container>
+      </b-row>
+    </b-container>
+  </cos-modal>
+</template>
+
+<script>
+import CosModal from "../CosModal"
+
+export default {
+  components: {
+    CosModal,
+  },
+}
+</script>
+
+<style scoped>
+.row {
+  padding: 5px 0;
+}
```

Comment:
Just FYI, Bootstrap has really useful spacing classes that can be used to avoid doing custom CSS like this. For example, you can just give the class `py-2` to have a similar padding added. See https://getbootstrap.com/docs/4.0/utilities/spacing/